x-pack/plugins/fleet/common/openapi/paths/agent_status.yaml

```diff
@@ -41,3 +41,13 @@ get:
       name: policyId
       in: query
       required: false
+    - schema:
+        type: string
+      name: kuery
+      in: query
+      required: false
+    - schema:
+        type: string
+      name: enrolledAt
```

Comment:
Do we have to add a new parameter here? can we use the kuery one directly?

Reply:
I removed the new parameter and I'm now using the existing `kuery`. Could you take a look again?